qvm-tools/qvm-ls

```diff
@@ -24,7 +24,7 @@
 from qubes.qubes import QubesVmCollection
 from qubes.qubes import QubesHost
 from qubes.qubes import QubesException
-from optparse import OptionParser
+from argparse import ArgumentParser
```

Comment:
I'm perfectly fine with this change. It's just about having it in separate commit.